loops-and-arrays.js

```diff
@@ -59,3 +76,13 @@ const squares = [30, 2, 8, 24, 11];
 
 
 
+console.log(squares);
+
+for (let i = 0; i < squares.length; i++) {
+
+
+    squares[i] = ("Het volume van " + squares[i] + " is " + squares[i] * squares[i] * squares[i]);
```

Comment:
Wist je dat je ook dit kon doen?
```suggestion
    squares[i] = ("Het volume van " + squares[i] + " is " + squares[i] ** 3);
```

Reply:
Mooi snelwegje! ik wist het niet idd!!